### PR TITLE
[FEAT]: Profile page data connected

### PR DESCRIPTION
**Pull request by shivanigangadharan**
Worked on Profile page and the following features :- 

- Logged in user's profile page connected to their data
- Implemented Follow/Unfollow
- Conditionally displayed followers/following/posts on profile page
- Navigated to home on logout

### CODE REVIEW

File: src/components/follow-them/follow-them.js [hunk 1]
```diff
@@ -1,9 +1,83 @@
...
+
+export default function FollowThem({ userObj }) {
+    const { fullName, username, following, uid, id, followers } = userObj;
+    const { user, setUser } = useAuth();
+
+    const addToFollowing = async () => {
+        try {
+            console.log(uid);
+            console.log(user.following[2]);
+            console.log(user.following[2] === uid)
```
Comment (devananth) [inline src/components/follow-them/follow-them.js]: If not required, you can *remove those console.logs*

**Review by devananth — CHANGES_REQUESTED** [review]
- Code looks well structured.
- Good Work

File: src/components/follow-them/follow-them.js [hunk 2]
```diff
@@ -1,9 +1,83 @@
...
+    }
+
+    const removeFromFollowers = async () => {
+        try {
+            const userRef = doc(db, `users/${id}`);
+            const res = await updateDoc(userRef, {
+                followers: arrayRemove(user.uid)
+            });
+        } catch (e) {
+            console.log(e)
```
Comment (sharathnair9999) [inline src/components/follow-them/follow-them.js:58]: handle errors with toast

**Review by sharathnair9999 — APPROVED** [review]

File: src/components/follow-them/follow-them.js (hunk 1 above)
Comment (shivanigangadharan) [reply src/components/follow-them/follow-them.js]: Done [2cac699](https://github.com/shivanigangadharan/social-media/pull/8/commits/2cac699d200222db146bddaf9481099ca56852be)

File: src/components/follow-them/follow-them.js (hunk 2 above)
Comment (shivanigangadharan) [reply src/components/follow-them/follow-them.js:58]: Sure, will do that in the following branch/PRs. Thanks for suggesting.

**Pull request merged**